### PR TITLE
Add a snippet for struct

### PR DESCRIPTION
**Pull request by srabuini**
Hey,

I've added a snippet for `struct`.

Please have a look and let me know.

Thanks!

### CODE REVIEW

File: snippets/language-crystal.cson [hunk 1]
```diff
@@ -230,6 +230,9 @@
   'module .. end':
     'prefix': 'mod'
     'body': 'module ${1:ModuleName}\n\t$0\nend'
+  'struct .. end':
+    'prefix': 'mod'
```
Comment (keplersj) [inline snippets/language-crystal.cson]: Gonna need to change the prefix. Maybe something like `struct`?

**Comment by keplersj** [comment]
Looks great! Just need to change the prefix and it'll be good to merge!

**Comment by srabuini** [comment]
Sorry about that! I realized my mistake and changed just my editor snippets. I should be checked before. My bad.

Fixed!

**Comment by keplersj** [comment]
Woot!

**Comment by keplersj** [comment]
I'll release it on APM soon!

**Pull request merged**